hub.go

```diff
@@ -0,0 +1,49 @@
+package main
```

Comment:
¿no sería mejor crear su propio paquete? Para que no todo este cargado en main y solo se importe donde se ocupe

Reply:
> The naming convention for Go package is to use the name of the system directory where we are putting our Go source files. Within a single folder, the package name will be same for the all source files which belong to that directory.

> When you build reusable pieces of code, you will develop a package as a shared library. But when you develop executable programs, you will use the package “main” for making the package as an executable program. The package “main” tells the Go compiler that the package should compile as an executable program instead of a shared library.

src: https://thenewstack.io/understanding-golang-packages/

Reply:
You were right. I updated the files and separated them into `./api` and `./socket`.